src/collection.js

```diff
@@ -760,9 +761,20 @@ export default class Collection {
       const resolution = strategy === Collection.strategy.CLIENT_WINS
         ? conflict.local
         : conflict.remote;
-      const updated = this._resolveRaw(conflict, resolution);
-      transaction.update(updated);
-      return updated;
+      const rejected = strategy === Collection.strategy.CLIENT_WINS
+        ? conflict.remote
+        : conflict.local;
+      let accepted;
+      if (resolution === null) {
```

Comment:
Maybe here I would add a comment to specify that it mainly applies to server wins (since ``conflict.local`` might never be null) and add a link to the piece of code in kinto-http that uses `error.details` of *412 responses*